toplevel/coqtop.ml

```diff
@@ -519,6 +518,8 @@ type custom_toplevel = {
 let coqtop_init ~opts extra =
   init_color opts;
   CoqworkmgrApi.(init !async_proofs_worker_priority);
+  if CList.is_empty extra then
```

Comment:
I think this check is not necessary; otherwise that's a good patch that could go even on `master`.

Reply:
Without this test, the behavior changes with respect to previous versions. Indeed, this would give the following output:
```
$ bin/coqtop -foo
Welcome to Coq machinette:/home/guillaume/src/coq,v8.9 (af858147b44b934f041f7d2f135046f41ef18577)
Don't know what to do with -foo
See -help for the list of supported options
```
instead of the expected output:
```
$ bin/coqtop -foo
Don't know what to do with -foo
See -help for the list of supported options
```